plugins/content_distribution/providers/cortex/lib/CortexApiDistributionEngine.php

```diff
@@ -407,8 +423,42 @@ protected function uploadVideo(KalturaCortexApiDistributionJobProviderData $apiD
 	{
 		try
 		{
-			$SystemIdentifier = $this->mediaUpload($apiDistributionJobProviderData->videoAssetFilePath);
+            $needDel = false;
+            $videoFilePath = $apiDistributionJobProviderData->videoAssetFilePath;
+            if (!$videoFilePath)
```

Comment:
add {} for all if statements here

Reply:
Fixed here - https://github.com/kaltura/server/pull/12079/commits/d26def18891e9adca0d378d5a70af1d68110a07a